### PR TITLE
Ds3 panel multiclick MacOS

### PR DESCRIPTION
**Pull request by scribe**
Ok, so the multi-select change we made to the local file browser should be applied to the DS3 file browser as well for consistency.

As long as I was fixing that, I took a stab at some low-hanging fixes.

### CODE REVIEW

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java [hunk 1]
```diff
@@ -272,37 +272,9 @@ protected void updateItem(final String item, final boolean empty) {
                 }
             });
 
-            sizeColumn.setCellFactory(c -> new TreeTableCell<Ds3TreeTableValue, Number>() {
+            sizeColumn.setCellFactory(c -> new NumberTreeTableCell());
 
-                @Override
```
Comment (DenverM80) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java]: nice, I do like to see this sort of refactoring to clean things up.

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java [hunk 2]
```diff
@@ -646,6 +618,38 @@ private void checkInterruptedJob(final String endpoint) {
...
+            if (empty || item == null) {
+                setText(null);
+            } else {
+                setText(FileSizeFormat.getFileSizeType(item.longValue()));
+            }
+        }
+
+    }
+
+    private static class TreeTableValueTreeTableCell extends TreeTableCell<Ds3TreeTableValue, Ds3TreeTableValue.Type> {
```
Comment (DenverM80) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java]: could we call this a ValueTreeTableCell, to be consistent with above?

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/savedSessionStore/SavedSessionStore.java [hunk 3]
```diff
@@ -142,11 +142,8 @@ private boolean isNewValuePresent(final SavedSession savedSession, final Session
         } else if (session.getProxyServer() != null) {
             return true;
         }
-        if (savedSession.getDefaultSession() != session.getDefaultSession()) {
-            return true;
-        }
+        return savedSession.getDefaultSession() != session.getDefaultSession();
```
Comment (DenverM80) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/savedSessionStore/SavedSessionStore.java]: (thumbsup)

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java [hunk 4]
```diff
@@ -602,4 +590,18 @@ private Path getSelectedLocalPath(final TreeTableRow<FileTreeModel> row) {
         }
         return localPath;
     }
+
+    private static class TableCell extends TreeTableCell<FileTreeModel, Number> {
```
Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java]: This looks almost identical to `TreeTableCell` in file `JobInfoPresenter.java` can we get rid of one of them?

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/Priority.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
+package com.spectralogic.dsbrowser.gui.util;
+
+/**
+ * Created by ericb on 7/17/17.
```
Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/Priority.java]: Remove this comment

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/Priority.java [hunk 6]
```diff
@@ -0,0 +1,14 @@
+package com.spectralogic.dsbrowser.gui.util;
```
Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/Priority.java]: The copyright is missing.

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/ValueTreeTableCell.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
+package com.spectralogic.dsbrowser.gui.util;
```
Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/ValueTreeTableCell.java]: Missing the copyright

Comment (scribe) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/ValueTreeTableCell.java]: Fixed in latest commit

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/Priority.java (hunk 6 above)
Comment (scribe) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/Priority.java]: Fixed in latest commit

**Pull request merged**